packages/common/src/store/ui/toast/slice.ts

```diff
@@ -20,6 +20,14 @@ const slice = createSlice({
       const toast = action.payload
       state.toasts.push(toast)
     },
+    manualClearToast: (state, action: DissmissToastAction) => {
+      const toastIdx = state.toasts.findIndex(
+        (t) => t.key === action.payload.key
```

Comment:
so cleanest imo would be to remove the toast element from the state, and then the toast component would see the element is no longer there, and animate out? this is a cool way to make it happen though.

Reply:
if we go with this implementation, would something like this work?
```
const toastToClear = state.toasts.find(....)
toastToClear.timeout = 0
```

Reply:
and just final callout, i think as much as we can, use full words for things, even known words that can be minimized, so instead of idx, use index. maybe would be a good thing to check with everyone on client, and possibly add to styleguide.

Reply:
I tried to simply remove the toast from state, but the toast elements are rendered from the toasts and immediately disappears if it is removed. I went with this pattern to have the toast listen for the change and dismiss itself bc it looked nicer. Open to alternative solutions to this though bc it does feel a bit unintuitive

Reply:
yeah makes sense, i think "react-spring" can handle it but maybe not react-native animations